src/webrtc/callFeed.ts

```diff
@@ -188,4 +189,8 @@ export class CallFeed extends EventEmitter {
             this.volumeLooper();
         }, POLLING_INTERVAL);
     }
+
+    public dispose(): void {
+        clearTimeout(this.volumeLooperTimeout);
```

Comment:
This dispose method just makes sure we don't keep the timeout alive after we've removed the call feed. I can split this out into another PR if we'd like.

Reply:
Moved to https://github.com/matrix-org/matrix-js-sdk/pull/1904